images/saige-qtl/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM australia-southeast1-docker.pkg.dev/analysis-runner/images/driver-r:1.1
+
+SHELL ["/bin/bash", "-eo", "pipefail", "-c"]
+RUN conda create -n RSAIGE r-essentials r-base=4.1.2 python=3.10 \
```

Comment:
`micromamba` will be much faster to resolve things than `conda`.

Can you please change this to use `micromamba install -y --prefix $MAMBA_ROOT_PREFIX ...` (see https://github.com/populationgenomics/analysis-runner/blob/24673e40894262513ab2bc79c5e9ac0996921cf2/driver/Dockerfile.r#L14-L24)? You don't need to create a new conda environment either, just install it in the base environment (since this is isolated in a Docker image anyway). Also best not to chain multiple `install` operations, but instead install everything at once. This way, the dependencies only need to be resolved once.

Reply:
yes should be done now!